resources/report_issue_user_settings.json

```
@@ -0,0 +1,126 @@
+{
+    "initialize": false,
+    "pythonPath": false,
+    "onDidChange": false,
+    "defaultInterpreterPath": false,
+    "defaultLS": {
+        "defaultLSType": true
+    },
+    "showStartPage": true,
+    "downloadLanguageServer": true,
+    "jediPath": false,
+    "jediMemoryLimit": false,
+    "envFile": false,
+    "venvPath": "placeholder",
+    "venvFolders": false,
+    "condaPath": false,
+    "pipenvPath": false,
+    "poetryPath": false,
+    "devOptions": false,
+    "disableInstallationChecks": false,
+    "globalModuleInstallation": false,
+    "autoUpdateLanguageServer": false,
+    "languageServer": true,
+    "languageServerIsDefault": false,
+    "logging": false,
+    "useIsolation": false,
+    "changed": false,
+    "_pythonPath": false,
+    "_defaultInterpreterPath": false,
+    "workspace": false,
+    "workspaceRoot": false,
+    "linting": {
+        "enabled": true,
+        "cwd": true,
+        "Flake8Args": true,
```

```suggestion
        "flake8Args": true,
```
Should be lowercased.